lib/backend/model/ippool.go

```diff
@@ -27,15 +27,15 @@ import (
 )
 
 var (
-	matchPool = regexp.MustCompile("^/?calico/v1/ipam/v./pool/([^/]+)$")
-	typePool  = reflect.TypeOf(Pool{})
+	matchIPPool = regexp.MustCompile("^/?calico/v1/ipam/v./pool/([^/]+)$")
```

Comment:
this shouldn't matter but, are we not changing the etcd key path from pool to ippool?